test/integration/pack/QtiItemPackerTest.php

```diff
@@ -86,6 +86,29 @@ public function testPackingInvalidQtiItem()
         $itemPackerMock->packItem(new core_kernel_classes_Resource('foo'), 'en-US', $this->getDirectoryStorage());
     }
 
+    /**
+     * Test packing an item where QTI content isn't valid, but validation skipped
+     */
+    public function testPackingInvalidQtiItemWithoutValidation()
+    {
+        $filename = dirname(__FILE__) . '/../samples/wrong/notvalid_associate.xml';
+        $this->assertTrue(file_exists($filename));
+
+        $itemPackerMock = $this
```

Comment:
I strongly disagree with mocking class that we want to test.

Reply:
We need to mock it, because we need to provide return value for `$this->getXmlByItem()` method, which related to tested class.